### DISCUSSION TITLE
ESP32 Module with USB-C and ext. Antenna

### DISCUSSION THREAD

**Discussion by CheckmaHardware** [tbnobody/OpenDTU#463] [Q&A]
Hi!

- does anybody know a ESP32 Module with USB-C and external Antenna Connector that ist usable for OpenDTU?

The only Module I have found is the "Adafruit 5438 Feather ESP32 V2 w.FL Antenne" 

https://botland.de/mit-arduino-kompatible-kacheln-adafruit/21468-feather-esp32-v2-wfl-antenne-wifi-modul-kompatibel-mit-arduino-adafruit-5438.html

- I think this Module should be compatible as it uses a standard ESP32 Controller or is there any arguments against that?

**Comment by martin8310** [comment]
it doesn't matter what the module is called, it just has to be an ESP32 and the PINs has to be present.

https://www.ebay.de/itm/204101407105

**Reply by koerri** [reply]
That ebay items seems to be missing an antenna connector though.
You might have to "hack" it! :-)
https://community.home-assistant.io/t/how-to-add-an-external-antenna-to-an-esp-board/131601
